docs/sections/user_guide/cli/drivers/chgres_cube.rst

```diff
@@ -1,6 +1,8 @@
 ``chgres_cube``
 ===============
 
+.. include:: /shared/idempotent.rst
```

Comment:
Tangentially related: See below for the text of this note but, until we potentially implement something like the discussed-in-Slack `--reset` switch to allow drivers to recreate their run directory, users may benefit from being informed about the idempotent nature of the drivers and the fact that they'll need to remove items they want the driver to recreate.